lib/razor/data/hook.rb

```diff
@@ -149,8 +149,15 @@ def validate
   end
 
   def log(params = {})
-    cursor = Razor::Data::Event.order(:timestamp).order(:id).reverse.
-        where(hook_id: id).limit(params[:limit], params[:start])
+    cursor = Razor::Data::Event.order(:timestamp).order(:id).
+        where(hook_id: id)
+    total = cursor.count if cursor.respond_to?(:count)
+    if params[:start].nil? and params[:limit] and !total.nil?
+      # We have a request for a limited list of facts without a starting
+      # value. Take from the end so the latest entries are included.
+      params[:start] = [total - params[:limit], 0].max
```

Comment:
This may be considered a break of the API. Probably a better way to deal with this is to add metadata properties to the previous endpoint or something. I'm not clear on the best approach here.